test_type/shard.js

```diff
@@ -322,14 +326,32 @@ async function clear(testList) {
   console.log(`- Finish to cleanup data`);
 }
 
+async function checkAndWriteMonitoringInfo(benchmarkConfig, startTime, endTime, testResult) {
```

Comment:
minor) I think the name is a little misleading since it's getting info but there's no "checking (validating)" or "writing" of info. How about renaming this to `getMonitoringInfo()` and changing the name of current `getMonitoringInfo()` to something like `getMonitoringInfoFromGoogleCloud()`?

Reply:
Looks better. I'll change it!